Tools/CISupport/ews-build/steps.py

```diff
@@ -213,17 +214,32 @@ def get_reviewers(self, pr_number, repository_url=None):
         if not api_url:
             return []
 
-        reviews_url = f'{api_url}/pulls/{pr_number}/reviews'
-        content = self.fetch_data_from_url_with_authentication_github(reviews_url)
-        if not content:
-            return []
+        reviews = []
+        reviews_url = f'{api_url}/pulls/{pr_number}/reviews?per_page={self.LIMIT}'
+        page = 1
+        while page < 10:
+            content = self.fetch_data_from_url_with_authentication_github(f'{reviews_url}&page={page}')
+            if not content:
+                break
+            response_content = content.json() or []
+            if not isinstance(response_content, list):
+                break
+            reviews += response_content
+            if len(response_content) >= self.LIMIT:
```

Comment:
why do we need this check?

Reply:
We're asking for 100 on the page. If the page has less than 100, we can safely assume this is the last page. Most of the time, we won't even make a second request.

Reply:
Seems like the comparison need to be inverted than. Should be `if len(response_content) < self.LIMIT:`